src/transfer/transfer.page.tsx

```diff
@@ -0,0 +1,145 @@
+import React, { useState, useEffect } from "react";
+import {
+  Divider,
+  Grid,
+  CircularProgress,
+  Paper,
+  Box,
+  TableContainer,
+  Table,
+  TableBody,
+  TableHead,
+  TableRow,
+  Button,
+  TableFooter,
+  TablePagination,
+} from "@mui/material";
+import { Add } from "@mui/icons-material";
+import { users } from "../data/user";
+import { StyledTableCell, StyledTableRow } from "./transfer.styles";
+import { useNavigate } from "react-router-dom";
+
+export const Transfer = () => {
+  const [page, setPage] = React.useState(0);
+  const [rowsPerPage, setRowsPerPage] = React.useState(5);
+  const handleChangePage = (
+    event: React.MouseEvent<HTMLButtonElement> | null,
+    newPage: number
+  ) => {
+    setPage(newPage);
+  };
+  const handleChangeRowsPerPage = (
+    event: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>
+  ) => {
+    setRowsPerPage(parseInt(event.target.value, 10));
+    setPage(0);
+  };
+
+  const [beneficiaries, setBeneficiaries] = useState([
+    {
+      accountNumber: "",
+      firstName: "",
+      lastName: "",
+      bankName: "",
+    },
+  ]);
+  const [loading, setLoading] = useState(true);
+  const navigate = useNavigate();
+  useEffect(() => {
+    const timeout = setTimeout(() => {
+      setLoading(false);
+      setBeneficiaries(users[0].beneficiaries.reverse());
+    }, 1200);
+    return () => {
+      clearTimeout(timeout);
+    };
+  }, []);
+
+  return (
+    <Paper elevation={3} variant="elevation">
+      <Box p={3}>
+        {loading && (
+          <Grid container alignItems="center" justifyContent="center">
+            <Grid item>
+              <CircularProgress />
+            </Grid>
+          </Grid>
+        )}
+        {!loading && (
+          <>
+            <Grid container justifyContent="space-between" alignItems="center">
+              <h2>Transfer Funds</h2>
+              <Button endIcon={<Add />} variant="contained">
+                Add Beneficiary
+              </Button>
+            </Grid>
+            <Divider />
+            <TableContainer component={Paper}>
+              <Table>
+                <TableHead>
+                  <TableRow>
+                    <StyledTableCell align="center">Name</StyledTableCell>
+                    <StyledTableCell align="center">Bank</StyledTableCell>
+                    <StyledTableCell align="center">
+                      Account Number
+                    </StyledTableCell>
+                    <StyledTableCell align="center">Actions</StyledTableCell>
+                  </TableRow>
+                </TableHead>
+                <TableBody>
+                  {beneficiaries.map(
+                    ({ accountNumber, firstName, lastName, bankName }) => (
+                      <StyledTableRow key={accountNumber}>
```

Comment:
Please use a proper id as key or use index instead.

Reply:
accountNumber is a proper Id and is unique